src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValidationStep.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives
+{
+    public static class ValidationStep
```

Comment:
Would this be better as an enum?

Reply:
Paul added it as a string.  Shall ask him

Reply:
It's because its a value stored in the ValidationResult table, and they wanted it to be readable for reporting (I assume)

Reply:
If you decorate the EF model with the Column attribute, it will be stored as a string instead of the numeric enum value - example here:
https://github.com/SkillsFundingAgency/das-employer-incentives/blob/00b579777c90969e3de5cc3760d28b588bdb5d34/src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/Models/PendingPayment.cs#L22